applications/IgaApplication/tests/NodeCurveGeometry3DTests.py

```diff
@@ -0,0 +1,109 @@
+from KratosMultiphysics import *
```

Comment:
name of python scripts should not be CammelCase

Reply:
`node_curve_geometry_3d_tests.py`?

Reply:
Better :D